lib/errors.js

```diff
@@ -264,7 +242,8 @@ exports.ConnectionRefusedError = ConnectionRefusedError;
 /**
  * Thrown when a connection to a database is refused due to insufficient privileges
  * @extends ConnectionError
- * @constructor
+ * @memberof Errors
+ * @memberof Sequelize
```

Comment:
This is the only Error class that is also declared as `memberof Sequelize`. I thought all Error classes are memberof Sequelize?